### PR TITLE
Enables Water Coolers to be movable

### PR DESCRIPTION
**Pull request by SamHPurp**
Simply wrench them. :tada: now you can re-design Escape to your hearts content.

:cl: Purpose2
add: Water coolers have been fitted with better bolts, enabling them to be loosened and moved.
/ :cl:

### CODE REVIEW

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -223,6 +223,34 @@
...
 	paper_cups--
 
+/obj/structure/reagent_dispensers/water_cooler/attackby(obj/item/weapon/W, mob/living/user, params)
+	add_fingerprint(user)
+	user.changeNext_move(CLICK_CD_MELEE)
+	if(iswrench(W))
+		if(anchored)
+			playsound(loc, W.usesound, 100, 1)
+			user.visible_message("[user] is loosening [src]'s floor casters.", \
+								 "<span class='notice'>You are loosening [src]'s floor casters...</span>")
```
Comment (uraniummeltdown) [inline code/modules/reagents/reagent_dispenser.dm]: for consistency I think this should be "You start loosening..."

File: code/modules/reagents/reagent_dispenser.dm [hunk 2]
```diff
@@ -223,6 +223,34 @@
...
+			if(do_after(user, 40 * W.toolspeed, target = src))
+				if(!loc || !anchored)
+					return
+				user.visible_message("[user] loosened [src]'s floor casters!", \
+									 "<span class='notice'>You loosen [src]'s floor casters!</span>")
```
Comment (uraniummeltdown) [inline code/modules/reagents/reagent_dispenser.dm]: I would drop the ! here for consistency again

File: code/modules/reagents/reagent_dispenser.dm [hunk 3]
```diff
@@ -223,6 +223,34 @@
...
+				anchored = 0
+		else
+			if(!isfloorturf(loc))
+				user.visible_message("<span class='warning'>A floor must be present to secure [src]!</span>")
+				return
+			playsound(loc, W.usesound, 100, 1)
+			user.visible_message("[user] is securing [src]'s floor casters...", \
+								 "<span class='notice'>You are securing [src]'s floor casters...</span>")
```
Comment (uraniummeltdown) [inline code/modules/reagents/reagent_dispenser.dm]: start instead of are

**Comment by Ralta** [comment]
YEEEEEEEEEEEEEEEES. Thank you.

**Comment by Spacemanspark** [comment]
Doing God's work

**Pull request merged**